schemas/test/en/test_progress_value_source_repeating_sections_chained_dependencies.json

```diff
@@ -0,0 +1,491 @@
+                                                    "value": "Yes",
+                                                    "action": {
+                                                        "type": "RemoveListItemAndAnswers"
+                                                    }
+                                                },
+                                                {
+                                                    "label": "No",
+                                                    "value": "No"
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                }
+                            },
+                            "summary": {
+                                "title": "Household members",
+                                "item_title": {
+                                    "text": "{person_name}",
+                                    "placeholders": [
+                                        {
+                                            "placeholder": "person_name",
+                                            "transforms": [
+                                                {
+                                                    "arguments": {
+                                                        "delimiter": " ",
+                                                        "list_to_concatenate": [
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "second-first-name"
+                                                            },
+                                                            {
+                                                                "source": "answers",
+                                                                "identifier": "second-last-name"
+                                                            }
+                                                        ]
+                                                    },
+                                                    "transform": "concatenate_list"
+                                                }
+                                            ]
+                                        }
+                                    ]
+                                }
+                            }
+                        },
+                        {
+                            "type": "Question",
+                            "id": "second-question-block",
+                            "question": {
+                                "id": "second-question",
+                                "title": "Question",
+                                "description": ["The next question is used as a dependency in the repeating sections."],
```

Comment:
I don't think it is? can't see a dependency on second-question-block

Reply:
Removed this, was copied from the non chained version of the schema 👍